c7n/resources/apigw.py

```diff
@@ -68,6 +68,25 @@ def get_resources(self, resource_ids):
 
 @RestAccount.action_registry.register('update')
 class UpdateAccount(BaseAction):
+    """Update the cloudwatch role associated to a rest account
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: reduce-account-burst-limie
```

Comment:
policy name does not related to policy content for the example also s/limie/limit